admin/src/routes/router.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+
+import { BurgersPage } from '../screens/burgers';
+import { IngredientsPage } from '../screens/ingredients';
+import { OrdersPage } from '../screens/orders';
+
+import { ROUTES } from '@constants/routes';
+
+export const Router: React.FC = () => {
```

Comment:
create folder `app` inside `screens`. then move `app.tsx`, `app.css` and this file (rename into `app.router.tsx`) there